index.html

```diff
@@ -2,35 +2,81 @@
+            <li class="nav-item"><i class="fa-brands fa-github"></i><a href="https://github.com/MarcinL09" target="_blank">github.com/MarcinL09</a></li>
         </ul>
-
-
-
     </nav>
     <main>
-        <h3 class="exp">Experience</h3>
+        <h3>Experience</h3>
+        <section class="exp">
+            <div class="left-side"><p>2020 - today</p></div>
+            <div class="right-side"><p><strong>Tile installer</strong> at <strong>Vivia Service</strong><br>Laying and setting tiles to create decorative wall, mural and floor designs. Cutting and fitting tiles around obstacles and openings using hand and power cutting tools. Preparing, measuring and marking surfaces to be covered.</p></div>
+        </section>
+        <section class="exp">
+            <div class="left-side"><p>2018 - 2020</p></div>
+            <div class="right-side"><p><strong>Assembly operator</strong> at <strong>Polentex</strong><br>Following operative guidelines to build a product for a company based on blueprints and instructions.</p></div>
+        </section>
+        <section class="exp">
+            <div class="left-side"><p>2013 - 2015</p></div>
+            <div class="right-side"><p><strong>Assembly operator</strong> at <strong>Xylem (Let’s Solve Water)</strong><br>
+                Working to strict safety and quality requirements, help to achieve daily
+                production requirements in terms of quality and quantity to meet customer
+                expectations and requirements, optimising efficiency and maintaining
+                operational excellence. Perform many operations within the process with
+                some supervision.</p></div>
+        </section>
+        <section class="exp">
+            <div class="left-side"><p>2012 - 2013</p></div>
+            <div class="right-side"><p><strong>Motorboats builder</strong> at <strong>Delphia Yachts Kot</strong><br>Reading and interpreting blueprints to build ships and boats. Makes repairs as necessary. Uses a variety of tools, instruments, and materials to create the boat.</p></div>
+        </section>
+        <section class="exp">
+            <div class="left-side"><p>2009 - 2010</p></div>
+            <div class="right-side"><p><strong>Automatiste</strong> at <strong>MSG</strong><br>Designing database systems. Analyzing the needs of an organization and produce an effective database system according to their needs and specifications.</p></div>
+        </section>
+        <h4>Education</h4>
+        <section class="exp">
+            <div class="left-side"><p>2004 - 2009</p></div>
+            <div class="right-side"><p>Olsztyn University of Warmia and Mazury</p></div>
+        </section>
+        <h4>Skills</h4>
+        <section class="exp">
+            <div class="skills">
+                <ul>
+                    <li>Good knowledge of how to level floors for tile surface applications</li>
```

Comment:
The English tenses are a bit mixed up here.
- "Experience surfacing and sanding floor surfaces",
- "Achieves excellent and beautiful tiling results",
- "applied grout and sealants"
Try making them a bit more consistent.